src/app/(sidebar)/smart-contracts/contract-explorer/components/ContractSpec.tsx

```diff
@@ -35,6 +35,9 @@ export const ContractSpec = ({
   const isXdrInit = useIsXdrInit();
   const queryClient = useQueryClient();
 
+  const [selectedLanguage, setSelectedLanguage] =
```

Comment:
question on naming - should we use `language` or `format`? we say `format` on [getEvents](https://lab.stellar.org/endpoints/rpc/get-events?$=network$id=testnet&label=Testnet&horizonUrl=https:////horizon-testnet.stellar.org&rpcUrl=https:////soroban-testnet.stellar.org&passphrase=Test%20SDF%20Network%20/;%20September%202015;;) (slight difference is that XDR format - `base64` or `json`)